server/src/mappers/store/InventoryItemMapper.ts

```diff
@@ -0,0 +1,28 @@
+import { InventoryItem } from "../../models/store/inventoryItem";
+
+export class InventoryItemMappper {
+  public static toDomain(raw: any): InventoryItem[] {
```

Comment:
Is there a benefit to having it be a static function instead of just
``` ts
export default function toDomain(raw: any): InventoryItem[] {
```

Reply:
Nope, just not used to making functions without a class I guess.

Reply:
In that case I would suggest going with exported functions, but it's not a deal breaker either way.